service/arangod_config_builder.go

```diff
@@ -202,19 +202,30 @@ func createArangodArgs(log zerolog.Logger, config Config, clusterConfig ClusterC
 			optionPair{"--foxx.queues", "true"},
 			optionPair{"--server.statistics", "true"},
 		)
+		if config.AdvertisedEndpoint != "" {
```

Comment:
I'll probably move this block out of the switch and wrap it in an `if serverType == ServerTypeCoordinator || serverType == ServerTypeResilientSingle { ... }`

This avoids code duplication.